src/components/carData/carData.java

```diff
@@ -0,0 +1,118 @@
+package components.carData;
+
+/**
+ * Represents a car and manages its information including make, model, year, and
+ * mileage.
+ */
+public class carData implements carDataKernel, carDataSecondary {
+    private String make;
+    private String model;
```

Comment:
Same goes for model. I try to avoid storing strings (especially meant to track categorical data) when possible.